frontend/src/styles/global.scss

```
@@ -779,3 +779,27 @@ body {
         border-radius: var(--radius);
     }
 }
+
+.loading-overlay {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: $bg_charcoal;
+    text-align: center;
+    min-height: 6rem;
+    opacity: 0.7;
```

```suggestion
    opacity: 0.5;
```
Found the level of darkness a bit out of style for us, 0.5 would be more typical. Connected to this is that the spinner should always have `opacity: 1` – it looks a bit ghostly when it has the same transparency as the overlay's background.